modules/tinymce/src/plugins/autolink/test/ts/browser/AutoLinkPluginTest.ts

```diff
@@ -27,34 +35,32 @@ UnitTest.asynctest('browser.tinymce.plugins.autolink.AutoLinkPluginTest', (succe
     editor.setContent('<p>' + modifiedurl + '</p>');
     LegacyUnit.setSelection(editor, 'p', modifiedurl.length);
     KeyUtils.type(editor, ')');
-    Assert.eq('Create a link of an eclipsed url', `<p>(<a href="${expectedUrl || url}">${url + dot}</a>)</p>`, editor.getContent());
+    assert.equal(editor.getContent(), `<p>(<a href="${expectedUrl || url}">${url + dot}</a>)</p>`, 'Create a link of an eclipsed url');
```

Comment:
Should we be using `TinyAssertions.assertContent()` here?

Reply:
Fixed